src/main/java/com/softserveinc/ita/pageobjects/FollowLinkPage.java

```diff
@@ -0,0 +1,34 @@
+package com.softserveinc.ita.pageobjects;
+
+import Elements.*;
+import org.openqa.selenium.By;
+
+public class FollowLinkPage extends AbstractBasePage {
+
+    private final Input searchInput = new Input(By.name("search"));
+    private final Button searchButton = new Button(By.xpath("//button[contains (@class, 'search-form__submit')]"));
+    private final Text searchResult = new Text(By.xpath("//p[contains (@class, 'catalog-selection__label')]"));
+    private final Ul lots = new Ul(By.xpath("//ul[contains (@class, 'catalog-grid')]"));
+
+    public FollowLinkPage() {
```

Comment:
according to the changes I have proposed, there will be such changes, also in the elements

```
    public FollowLinkPage(final WebDriver driver) {
        super(driver);
    }

    @Override
    public boolean isPageOpened() {
        return new Img(driver, By.xpath("//img[@alt='Rozetka Logo']")).isElementsDisplayed();
    }
```